app/container/template.hbs

```diff
@@ -103,10 +103,9 @@
 
     <div class="mt-20">
       {{container/form-scheduling
-        initialHostId=model.container.requestedHostId
-        initialLabels=model.container.labels
+        scheduling=model.container.scheduling
+        initialHostId=model.container.scheduling.nodeId
```

Comment:
`scheduling.node.nodeId`

Reply:
the type of nodeId is `reference[node]` via (`/v3/projects/project-${id}/schemas/nodeScheduling`). 
Should I use something like `scheduling.node.nodeId.id` here?